R/build.R

```diff
@@ -87,6 +87,7 @@ build_setup <- function(path, dest_path, binary, vignettes, manual, clean_doc, a
 
   if (needs_compilation) {
     update_registration(path, compile_attributes, register_routines, quiet)
+    withr:::local_makevars(compiler_flags(debug = FALSE), assignment = "+=", .local_envir = env)
```

Comment:
`local_makevars` was I think inadvertently not exported, I need to do a withr release soon anyway, so we can export it then.